generate.xml

```diff
@@ -1786,6 +1795,7 @@
       </build>
       <build name="bitcoin-protocol" github="libbitcoin" repository="libbitcoin-protocol" branch="master" parallel="true" >
         <option value="${with_boost}" />
+        <option value="${with_mbedtls}" />
```

Comment:
`mbedtls` follows `png` not `boost` in its pattern and population of parameter.  This option communicates an intent that the parameter is computed or manipulated by the install script and as that is not the case for `mbedtls` it is unnecessary.  Its presence doesn't appear to produce a problem, just noise.